dev_guide/builds/triggering_builds.adoc

```diff
@@ -278,7 +278,13 @@ $ oc describe bc <name>
 ----
 
 If the above command does not display any webhook URLs, then no webhook trigger
-is defined for that build configuration.
+is defined for that build configuration. In order to define a new webhook trigger,
+use `oc set triggers` with a `--from` flag. For example, to create a GitHub
+webhook, use `--from-github`:
+
+----
+$ oc set triggers bc <name> --from-github
+----
```

Comment:
I agree we want it somewhere, but probably not here.

i'd suggest putting the note here, in the overview:
https://docs.openshift.org/latest/dev_guide/builds/triggering_builds.html#webhook-triggers

I assume the web console buildconfig creation behavior will be the same (@jwforres), so the note should mention that too.  (Otherwise I would have suggested putting the note in the new-app doc section).